DiceKit/Die.swift

```diff
@@ -36,6 +36,13 @@ public struct Die: Equatable {
     
 }
 
+/**
```

Comment:
I've been inconsistent myself, but let's use `///` style everywhere for doc comments (the last holdout is in Die.swift, which we should cleanup sometime).